queries.yaml

```diff
@@ -516,32 +516,24 @@ post_process:
         - shield_text
         - all_networks
         - all_shield_texts
-        - bicycle_network
-        - bicycle_shield_text
-        - all_bicycle_networks
-        - all_bicycle_shield_texts
-        - all_walking_networks
-        - all_walking_shield_texts
         - bus_network
         - bus_shield_text
-        - all_bus_networks
-        - all_bus_shield_texts
       where: >-
         (kind == 'rail' and zoom < 15) or
         (kind == 'minor_road' and zoom < 14) or
         (kind == 'major_road' and zoom <  7)
-  # previous tests assume we keep walking network information until zoom 11
   - fn: vectordatasource.transform.drop_properties
     params:
       source_layer: roads
       start_zoom: 0
-      end_zoom: 10
+      end_zoom: 13
       properties:
-        - walking_network
-        - walking_shield_text
```

Comment:
This means that walking networks will be kept on the roads until zoom 7, but the earliest `min_zoom` for a walking route is zoom 9. This could block opportunities for merging roads at low zooms. Would we display walking data at z7-8? If not, we should remove this data.